docs/cookbook/basic-data-import.md

```diff
@@ -0,0 +1,481 @@
+[`Vat`](../../packages/framework/src/Model/Pricing/Vat/Vat.php) object
+in [`ProductData`](../../packages/framework/src/Model/Product/ProductData.php).
+So we will extend [`VatRepository`](../../packages/framework/src/Model/Pricing/Vat/VatRepository.php)
+and [`VatFacade`](../../packages/framework/src/Model/Pricing/Vat/VatFacade.php) and implement appropriate methods.*
+
+#### 3.6 - Extend [`VatRepository`](../../packages/framework/src/Model/Pricing/Vat/VatRepository.php) and implement method `getVatByPercent()` in order to load [`Vat`](../../packages/framework/src/Model/Pricing/Vat/Vat.php) by percent
+```php
+// Shopsys/ShopBundle/Model/Pricing/Vat/VatRepository.php
+
+namespace Shopsys\ShopBundle\Model\Pricing\Vat;
+
+use Shopsys\FrameworkBundle\Model\Pricing\Vat\VatRepository as BaseVatRepository;
+
+class VatRepository extends BaseVatRepository
+{
+    /**
+     * @param int $percent
+     * @return \Shopsys\FrameworkBundle\Model\Pricing\Vat\Vat
+     */
+    public function getVatByPercent($percent)
+    {
+        $vat = $this->getVatRepository()->findOneBy(['percent' => $percent]);
+
+        if ($vat === null) {
+            throw new \Shopsys\FrameworkBundle\Model\Pricing\Vat\Exception\VatNotFoundException('Vat with ' . $percent . '% not found.');
+        }
+
+        return $vat;
+    }
+}
+```
+
+Add information about the entity extension into the container configuration in [`services.yml`](../../project-base/src/Shopsys/ShopBundle/Resources/config/services.yml)
+```yaml
+    Shopsys\FrameworkBundle\Model\Pricing\Vat\VatRepository: '@Shopsys\ShopBundle\Model\Pricing\Vat\VatRepository'
+```
+
+**Warning: The method throws an exception when [`Vat`](../../packages/framework/src/Model/Pricing/Vat/Vat.php) object is not found by given percent value.
+Do not forget to handle it (e.g. skip the product data processing and log the exception).**
+
+#### 3.7 - Extend [`VatFacade`](../../packages/framework/src/Model/Pricing/Vat/VatFacade.php) and implement method `getVatByPercent()` in it
+```php
+// Shopsys/ShopBundle/Model/Pricing/Vat/VatFacade.php
+
+namespace Shopsys\ShopBundle\Model\Pricing\Vat;
+
+use Shopsys\FrameworkBundle\Model\Pricing\Vat\VatFacade as BaseVatFacade;
+
+class VatFacade extends BaseVatFacade
+{
+    /**
+     * @param int $percent
+     * @return \Shopsys\FrameworkBundle\Model\Pricing\Vat\Vat
+     */
+    public function getVatByPercent($percent)
+    {
+        return $this->vatRepository->getVatByPercent($percent);
+    }
+}
+```
+
+Add information about the entity extension into the container configuration in [`services.yml`](../../project-base/src/Shopsys/ShopBundle/Resources/config/services.yml)
+```yaml
+    Shopsys\FrameworkBundle\Model\Pricing\Vat\VatFacade: '@Shopsys\ShopBundle\Model\Pricing\Vat\VatFacade'
+```
+
+### Step 4 - Everything is done, you can run it now
+#### 4.1 Run `phing cron-list` in console
+As an output, you will get a list of all available cron modules.
+
+`
```

Comment:
You should use three backticks here, not one :slightly_smiling_face: I am not able to make such a suggestion for some reason :slightly_smiling_face: